stream-chat-android/src/main/java/com/getstream/sdk/chat/style/TextStyle.kt

```diff
@@ -9,8 +9,8 @@ import com.getstream.sdk.chat.ChatUI
 public class TextStyle {
     private companion object {
         const val UNSET_SIZE = -1
-        const val UNSET_COLOR = -1
-        const val UNSET_HINT_COLOR = -1
```

Comment:
-1 is a decimal representation of white color (#ffff) resulting in the wrong color here and there in our TextViews.

Reply:
![aaaaa](https://media.giphy.com/media/lb5nMu0UIh4YM/giphy.gif)